packages/techdocs-cli/.snyk

```diff
@@ -0,0 +1,17 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.22.1
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-JS-BROWSERSLIST-1090194:
+    - '*':
+        reason: Developer tools are not a valid target for ReDoS attacks
+        expires: 2022-05-20T00:00:00.000Z
+        created: 2021-11-20T00:00:00.000Z
+
+  SNYK-JS-IMMER-1540542:
```

Comment:
Hey can you double check if this one even applies anymore?

Reply:
@freben , Looks like they're both still applicable. I ran the following to confirm:
```
git pull upstream master
cd packages/techdocs-cli
snyk test
```
Which outputs:
```
Testing /home/<redacted>/backstage/packages/techdocs-cli...
                                                                                                      
Tested 44 dependencies for known issues, found 3 issues, 3 vulnerable paths.
                                                                                                                                                                                                            
                                                   
Issues with no direct upgrade or patch:                                                                                                                                                                     
  ✗ Regular Expression Denial of Service (ReDoS) [Medium Severity][https://snyk.io/vuln/SNYK-JS-BROWSERSLIST-1090194] in browserslist@4.14.2
    introduced by react-dev-utils@11.0.4 > browserslist@4.14.2                                        
  This issue was fixed in versions: 4.16.5                                                            
  ✗ Prototype Pollution [Medium Severity][https://snyk.io/vuln/SNYK-JS-IMMER-1540542] in immer@8.0.1
    introduced by react-dev-utils@11.0.4 > immer@8.0.1                                                
  This issue was fixed in versions: 9.0.6                                                             
  ✗ Regular Expression Denial of Service (ReDoS) [Medium Severity][https://snyk.io/vuln/SNYK-JS-PROMPTS-1729737] in prompts@2.4.0
    introduced by react-dev-utils@11.0.4 > prompts@2.4.0                                              
  This issue was fixed in versions: 2.4.2        
  ```